### PR TITLE
[BREAKING] upgrade ember-macro-helpers to ^4.0.0

### PR DESCRIPTION
**Pull request by jelhan**
Outdated version of `ember-macro-helpers` breaks Ember 3.10. It was fixed in `v3.1` by https://github.com/kellyselden/ember-macro-helpers/pull/260.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "8"
```
Comment (jasonmit) [inline .travis.yml]: Is there a breaking change here that I need to be aware of?

Comment (jelhan) [reply .travis.yml]: If I recall correctly, ember-macro-helpers has dropped support for Node 6 due to upgrading Ember CLI. Most addons consider this a breaking change.

**Comment by Subtletree** [comment]
Anything preventing this from being merged? Let me know if I can help

**Comment by jasonmit** [comment]
@Subtletree no - just needs to be merged as a new major

**Comment by jasonmit** [comment]
Merged https://github.com/stefanpenner/ember-moment/pull/314 which resolves this (only reason I took that PR was that it bumped the `engine` in pkg.json also.

**Comment by jasonmit** [comment]
v8.0.0 was released

**Comment by Subtletree** [comment]
awesome ty!!

**Pull request closed without merging**